.github/workflows/run-mobile-tests.yml

```diff
@@ -0,0 +1,23 @@
+name: Run mobile tests
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  mobile:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 🏗 Setup repository
+        uses: actions/checkout@v3
+
+      - name: 🏗 Setup monorepo
+        uses: ./.github/actions/setup-monorepo
+
+      - name: Run mobile tests
+        working-directory: apps/mobile
+        run: pnpm run test
```

Comment:
EOF line 😄

Reply:
Need to setup my editor to do that for me